### ISSUE TITLE
matterircd 0.7 sends privmsg that cannot be parsed

### ISSUE THREAD

**Issue by jtdaugherty** [42wim/matterircd#53]
Hi,

I'm using matterircd 0.7 and it sent a message to my chat client that the client could not parse:

```
irc: too few arguments received from IRC server for command "privmsg" (received: 3 arguments, expected: at least 4)
irc: failed to parse command "PRIVMSG" (please report to developers):
:dwest!ysthswine3f8bgqqse38c1pg7c@https://mattermost.galois.com PRIVMSG #questions-and-feedback
```
I've spoken with the chat client authors (weechat) and they're claiming that this is not an RFC-compliant message.  (That issue is at https://github.com/weechat/weechat/issues/751.)

CC @dwest-galois

**Comment by 42wim** [comment]
I can't reproduce this in 0.8. How did this happen ? Did a user messaged you from mattermost (which version) ? Can you try matterircd 0.8 (and run with -debug) and try to reproduce it?

**Comment by jtdaugherty** [comment]
I think @dwest-galois messaged me from MatterMost while I was logged into Weechat. We've ugpraded to 0.8 since this happened, so I'll keep an eye out for it happening again. If we were to run with debugging enabled, what would that enable that we should pass on to you (e.g. log files, etc.)?

**Comment by jtdaugherty** [comment]
Ah! Fantastic! I'll keep an eye out either way, but thank you for looking into this.

**Comment by jtdaugherty** [comment]
(And I was wrong: we will upgrade to 0.8 tomorrow.)

**Comment by 42wim** [comment]
It's fixed in master now, but I suppose it didn't crash the weechat client ? just a log/warning message ?

**Comment by jtdaugherty** [comment]
Yeah, I only got the text I posted in the original report; the client did not crash.

**Comment by dwest-galois** [comment]
Is this going to be released in v0.8.1 or v0.9?  I'm just looking for an ETA so that I can plan updates accordingly.

**Comment by 42wim** [comment]
Sorry, no ETA yet. I'll do a v0.8.1 if there's a crashing bug, but this is not the case for this one. 
It depends on how many other bugs pop up.
